build_projects/dotnet-host-build/PublishTargets.cs

```diff
@@ -280,6 +281,23 @@ public static BuildTargetResult PublishCoreHostPackages(BuildTargetContext c)
         }
 
         [Target]
+        public static BuildTargetResult PublishSignedPackages(BuildTargetContext c)
+        {
+            string signedPackages = EnvVars.EnsureVariable("SIGNED_PACKAGES");
+            if (signedPackages.Equals("true"))
```

Comment:
I didn't notice this earlier, but there's also a `EnvVars.GetBool` that would do a more extensive check, should use that I think: https://github.com/dotnet/core-setup/blob/eca152525e19a41d489cd5845afa14982653cc0a/build_projects/shared-build-targets-utils/Utils/EnvVars.cs#L10. Also allows null/empty, so we'd only need the var on the one definition we want to do this step.